xonsh/color_tools.py

```diff
@@ -397,17 +397,19 @@ def color_dist(x, y):
     return math.sqrt((x[0]-y[0])**2 + (x[1]-y[1])**2 + (x[2]-y[2])**2)
 
 
-def find_closest_color(x, pallette):
-    return min(sorted(pallette.keys())[::-1],
-               key=lambda k: color_dist(x, pallette[k]))
+def find_closest_color(x, palette):
+    return min(sorted(palette.keys())[::-1],
+               key=lambda k: color_dist(x, palette[k]))
 
 
-def make_pallete(strings):
-    """Makes a color pallete from a colection of strings."""
-    pallette = {}
+def make_palette(strings):
+    """Makes a color palette from a collection of strings."""
+    palette = {}
     for s in strings:
         while '#' in s:
             _, t = s.split('#', 1)
             t, _, s = t.partition(' ')
-            pallette[t] = rgb_to_ints(t)
-    return pallette
+            palette[t] = rgb_to_ints(t)
+    return palette
+
+make_pallette = make_palette
```

Comment:
Nifty. I'm thinking of making 2 decorators, `deprecated` and `replaced_by`. I have `deprecated` pretty much done, and I'll have `replaced_by` call `deprecated`. We'll use `replaced_by` for this and will have `deprecated` available in case we ever want to gracefully deprecate a function in the future.